spec/quality_spec.rb

```diff
@@ -40,6 +40,22 @@ def check_for_extra_spaces(filename)
     "#{filename} has spaces on the EOL on lines #{failing_lines.join(", ")}"
   end
 
+  def check_for_expendable_words(filename)
+    failing_line_message = []
+    useless_words = /\b(actually|obviously|just|clearly|basically|really)\b/i
+    second_person_words = /\b(he|she)\b/i
+
+    File.readlines(filename).each_with_index do |line, number|
+      if word_found = useless_words.match(line)
+        failing_line_message << "#{filename} has '#{word_found}' on line #{number + 1}. Avoid using such words."
```

Comment:
Might want to label what kind of words 'such words' refers to? (ex. 'Avoid using these kinds of weak modifiers'). Otherwise, the people reading this error message can't really figure out what kind of words they're supposed to be avoiding.